CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 * Added support to run specific hooks in **pre-commit**. Use with `demisto-sdk pre-commit <hook>`.
 * **Breaking change**: Removed the command **run-unit-tests**. Use `demisto-sdk pre-commit pytest-in-docker` instead.
 * **Breaking change**: Removed the `--unit-test` argument in **pre-commit**. To skip unit tests, run with `--no-docker` or with `skip=pytest-in-docker`,
-
+* Transformed the PB118 validation into the updated validation format and rectified the validation for both new and modified files.
```

Comment:
Let's remove it for now, since the new validate is not officially out yet we don't want to announce anything related.

Reply:
OK, Done.